tests/shared/src/test/scala/cats/tests/EitherSuite.scala

```diff
@@ -369,6 +387,19 @@ class EitherSuite extends CatsSuite {
     assert(either.toEitherNel === (Either.right[NonEmptyList[String], Int](42)))
   }
 
+  test("toEitherA Left") {
+    val either = Either.left[String, Int]("oops")
+    assert(
+      either
+        .toEitherA[NonEmptyVector, String] === (Either.left[NonEmptyVector[String], Int](NonEmptyVector.one("oops")))
+    )
+  }
+
+  test("toEitherA Right") {
+    val either = Either.right[String, Int](42)
+    assert(either.toEitherA[NonEmptyVector, String] === (Either.right[NonEmptyVector[String], Int](42)))
+  }
```

Comment:
A minor nitpick – take or leave it. I see that these two cases are mostly copied-and-altered from one of the existing cases. However, it is usually preferable to avoid hardcoded values in tests and use generators-based tests instead (i.e. `forAll` and all stuff like that). It should be really easy to convert to, if you'd like to.